### ISSUE TITLE
Misaligned Icons

### ISSUE THREAD

**Issue by ainen** [Memmy-App/memmy#518]
**Describe the bug**
On the “Feed” and “Traverse” screens, the icons on the top right are too close together. 

**To Reproduce**
1. Open “Feeds” or “Traverse”
2. Observe the top right icons being too close 

**Expected behavior**
The icons should be properly spaced the same way they are within a post. 

**Screenshots**
![image](https://github.com/Memmy-App/memmy/assets/1300058/e73fe5be-48fe-4efe-ba49-d8d1f076a406)
![image](https://github.com/Memmy-App/memmy/assets/1300058/88e20f6e-6ee2-4718-aaf9-e5bbdd2f3ec4)

**Smartphone (please complete the following information):**
 - Device: iPhone 14 Pro
 - OS: iOS 17 Beta 3
 - Version 0.1.0 (11)

**Comment by ebits21** [comment]
I've had a hard time clicking the right icon because of this.
